index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>hello microverse</title>
```

Comment:
:clap: :clap: Good job! Descriptive titles makes our pages optimized for search engines to efficiently crawl and present to users.